Analyse_Birds.qmd

```diff
@@ -79,8 +81,21 @@ Import dataset
 data <- read.table("Bird_dataset.txt", header = TRUE, stringsAsFactors = TRUE)
 ```
 
+If the data isn't attached yet, please consider doing so
+
+```{r}
+# detach(data) # If you have an error message, it is likely because the data is already attached, to fix this, please detach the data and attach it again
+attach(data)
+```
+
 Charge the packages
+Download the spDataLarge package, unavailable on CRAN
+```{r}
+# Please, run this cell only once
+install.packages("spDataLarge", repos = "https://nowosad.github.io/drat/", type = "source")
+```
```

Comment:
It would be better if you can put an "if" loop in that cell, so that the package is only installed if it was not already here, and people don't have to worry about running the cell only once

Reply:
Modification done. When run, the cell will now install spDataLarge from the drat depot only if R emists a "package required" message

Reply:
It is better practice to udpate the website.yml file to install the package before running the quarto document.
please look at the .github/workflows/website.yml and change it accordingly